perl6.html.markdown

```diff
@@ -891,12 +891,12 @@ for ^5 {
 ## * Role/class phasers
 COMPOSE { "When a role is composed into a class. /!\ NOT YET IMPLEMENTED" }
 
-# They allow for cute trick or clever code ...:
-say "This code took " ~ (time - CHECK time) ~ "s to run";
+# They allow for cute tricks or clever code ...:
+say "This code took " ~ (time - CHECK time) ~ "s to compile";
 
 # ... or clever organization:
 sub do-db-stuff {
-  ENTER $db.start-transaction; # New transaction everytime we enter the sub
```

Comment:
Do we? There are no examples for `BEGIN`, `INIT`, `END`. I think it's clear what `ENTER` does and IMO the phaser isn't that useful to warrant an example.